app/assets/javascripts/blacklight/facet_load.js

```diff
@@ -0,0 +1,15 @@
+(function($) {
+  Blacklight.onLoad(function() {
```

Comment:
AFAICT even explicitly requiring the file does not help.  I don't think @houndci evaluates required files (but I could be wrong, that's just in my limited experience).

I think I have just let these go for the most part in the requests project because I could never figure out how to get hound to stop complaining.